Tests/XCUITests/DatabaseFixtureTest.swift

```diff
@@ -37,10 +37,6 @@ class DatabaseFixtureTest: BaseTestCase {
        navigator.goto(LibraryPanel_Bookmarks)
        waitForExistence(app.tables["Bookmarks List"], timeout: TIMEOUT_LONG)
 
-       let loaded = NSPredicate(format: "count == 1001")
-       expectation(for: loaded, evaluatedWith: app.tables["Bookmarks List"].cells, handler: nil)
-       waitForExpectations(timeout: TIMEOUT_LONG, handler: nil)
```

Comment:
@jjSDET Please let me know if this isn't the modification that we've discussed. 🙂

Reply:
@clarmso not sure why you have to remove that... without that expectation this test fails for me...

Reply:
Without these lines, the test fails on my end. 😕 I can add those lines again.

Reply:
what we would need is to increase the timeout there so that the test works, same for the history test instead of removing the expectation, increase the timeout

Reply:
Done!